tigerpath/majors_and_certificates/majors/NEU_2018.json

```diff
@@ -0,0 +1,392 @@
+                        "PSY 252: Social Psychology",
+                        "PSY/NEU 345: Sensation & Perception*",
+                        "PSY 254: Developmental Psychology",
+                        "PSY/NEU 338: Animal Learning & Decision Making*",
+                        "EEB 311: Animal Behavior",
+                        "EEB 313: Behavioral Ecology",
+                        "EEB 329: Sensory Ecology"
+                    ]
+                },
+                {
+                    "name": "Quantitative Thinking",
+                    "max_counted": 1,
+                    "min_needed": 1,
+                    "explanation": "",
+                    "double_counting_allowed": false,
+                    "completed_by_semester": 8,
+                    "req_list": [
+                        {
+                            "name": null,
+                            "max_counted": 1,
+                            "min_needed": 0,
+                            "explanation": "",
+                            "double_counting_allowed": false,
+                            "completed_by_semester": 8,
+                            "req_list": [   
+                                {
+                                    "name": null,
+                                    "max_counted": 1,
+                                    "min_needed": 0,
+                                    "explanation": "",
+                                    "double_counting_allowed": false,
+                                    "completed_by_semester": 8,
+                                    "course_list": [   
+                                        "COS 126"
+                                    ],   
+                                },
+                                {
+                                    "name": null,
+                                    "max_counted": 1,
+                                    "min_needed": 0,
+                                    "explanation": "",
+                                    "double_counting_allowed": true,
+                                    "completed_by_semester": 8,
+                                    "req_list": [
+                                        "ISC 231",
+                                        "ISC 232",
+                                        "ISC 233",
+                                        "ISC 234"
+                                    ],   
+                                }
+                            ],   
+                        },
+                        {
+                            "name": null,
+                            "max_counted": 1,
+                            "min_needed": 0,
+                            "explanation": "",
+                            "double_counting_allowed": false,
+                            "completed_by_semester": 8,
+                            "course_list": [   
+                                "SML 201"

```

Comment:
If it's either SML 201 or COS 126 or ISC, then why not combine SML 201 and COS 126 into a single course list?